core/src/main/scala/caliban/schema/Schema.scala

```diff
@@ -131,17 +131,19 @@ trait GenericSchema[R] extends SchemaDerivation[R] with TemporalSchema {
    * @param name name of the scalar type
    * @param description description of the scalar type
    * @param specifiedBy URL of the scalar specification
+   * @param directives The directives to add to the type
```

Comment:
```suggestion
   * @param directives the directives to add to the type
```
nitpicking just to be consistent with the other parameters 😄